lib/infrastructure/datasources/PairedStorage.dart

```diff
@@ -45,6 +55,13 @@ class PairedStorage {
       .firstWhere((element) => element.device.address == address)
       .device;
 
+  static Future<List<PebbleDevice>> getAll() async {
+    List<PebbleDevice> devices = [];
```

Comment:
This will probably work better:

`return (await _readCurrent()).map((e) => e.device).toList()`

Reply:
I believe this will work the same and fit the code style better?
```
static Future<List<PebbleDevice>> getAll() async  => (await _readCurrent())
      .map((e) => e.device).toList();
```